modules/labs-react/toast/stories/examples/RTL.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import {Toast} from '@workday/canvas-kit-labs-react/toast';
+import {checkIcon} from '@workday/canvas-system-icons-web';
+import {colors} from '@workday/canvas-kit-react/tokens';
+import {CanvasProvider, ContentDirection} from '@workday/canvas-kit-react';
+
+export const RTL = () => {
+  const handleClose = () => console.log('close button clicked');
+
+  return (
+    <CanvasProvider theme={{canvas: {direction: ContentDirection.RTL}}}>
+      <Toast mode="dialog">
+        <Toast.Icon icon={checkIcon} color={colors.greenApple400} />
+        <Toast.Body>Your workbook was successfully processed.</Toast.Body>
```

Comment:
Some of our other examples nest the message within `Toast.Message`. Here (and in a few other examples as well), the message is contained directly within `Toast.Body` -- should we be using `<Toast.Body><Toast.Message>...</Toast.Message></Toast.Body>` here as well?